test/ScriptCs.Engine.Roslyn.Tests/CSharpScriptInMemoryEngineTests.cs

```diff
@@ -43,12 +43,12 @@ public void ShouldExposeExceptionThrownByScriptWhenErrorOccurs()
             [Fact]
             public void ShouldExposeExceptionThrownByCompilation()
             {
-                var scriptEngine = new RoslynScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
+                var scriptEngine = new CSharpScriptInMemoryEngine(new ScriptHostFactory(), new TestLogProvider());
 
                 // Arrange
                 var lines = new List<string>
                 {
-                    "using Sysasdasdasdtem;"
+                    "Sysasdasdasdtem;"
```

Comment:
@glennblock I guess that was in answer to https://github.com/scriptcs/scriptcs/pull/1064#issuecomment-130801874 ?